### PR TITLE
AT_05.02.006 | New item | Create a new Pipeline

### PR DESCRIPTION
**Pull request by DariaMas7**
https://trello.com/c/jqvRo0vV/580-at0502006-new-item-create-a-new-pipeline

### CODE REVIEW

**Review by IrynaLyakhova — APPROVED** [review]

File: cypress/e2e/createNewPipeline.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress"/>
+describe('New item',()=>{
+    const nameOfPipeline = 'New Pipeline'
+    it('Create a new Pipeline',()=>{
+        cy.viewport(1920,1080)
```
Comment (Rom-T) [inline cypress/e2e/createNewPipeline.cy.js:5]: This setting is not needed. It is already registered in e2e

File: cypress/e2e/createNewPipeline.cy.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress"/>
+describe('New item',()=>{
```
Comment (Rom-T) [inline cypress/e2e/createNewPipeline.cy.js:2]: Please name the describe as the name of your file.

Comment (DariaMas7) [reply cypress/e2e/createNewPipeline.cy.js:2]: I am confused, I thought that in describe we write the name of Epic, and in it we write the name of the TC.
In this case if I write describe as an 'Create a new Pipeline' what should I write in it?

Comment (Rom-T) [reply cypress/e2e/createNewPipeline.cy.js:2]: Maria said in a lecture yesterday that we should name the describe like a file. We take this name from their User story, and if it is long, we shorten it.

**Review by YurySelivanov — APPROVED** [review]

**Review by IrynaLyakhova — APPROVED** [review]

**Review by IrynaLyakhova — APPROVED** [review]

**Review by Parizat-Ismailova — APPROVED** [review]

**Review by Parizat-Ismailova — APPROVED** [review]

**Review by AnnaDit1 — APPROVED** [review]

**Review by ArinaJur — APPROVED** [review]
ОТлично!

**Review by Alxndrpz — DISMISSED** [review]

**Review by IrynaLyakhova — APPROVED** [review]

**Review by Rom-T — APPROVED** [review]

**Pull request merged**